### ISSUE TITLE
More specific documentation and Trady test

### ISSUE THREAD

**Issue by IsaacFilarski** [lppkarl/Trady#82]
Hello there,

I'm a new dotnet developer and I'm trying to make use of your Framework but i see very limited documentation. The only test I've been running is the benchmark and I find some things unclear.

I can't even find a tutorial of how Trady works on youtube there's only literally just what you offer.

Are there any plans on making more documentation?

Thanks for all :)

Greetings.

**Comment by lppkarl** [comment]
@IsaacFilarski @SariSultan Actually, it is discussed several months ago in #66 but the only current document is the readme file. For additional documentation, I would need to see if time is available as I'm quite busy on my work now.

**Comment by IsaacFilarski** [comment]
@lppkarl @irperez Well I would like to know in detail how the rules work. It would be very helpful if you someday take an hour to record a tutorial of you using Trady and explaining/covering as much as you can.
I plan on posting my project with some limitations when I finish to help starters.

**Comment by rossspaul** [comment]
I agree, I'm struggling to do simple things like RSI(14) or ATR(20).  Setting up the data properly and passing that data to these functions is very confusing.  More examples on data setup (IOhlc) and calling the indicator functions please!  Great library but hard to use.

**Comment by MehdiAnis** [comment]
> What specifically would you like to see documentation on?
> 
> I started out feeling the same way, but after going through the unit tests, most of my questions were answered.

Test functions utilizes Extension methoids of Ienumerable<IOhlc> like, candles.Cci(20). In my case I would like to see some example of calling CCI function directly - 

var cciResults = new CommodityChannelIndexByTuple(Input, 14);

I do not know how to create the "Input" object. An example of creating IEnumerable<decimal High, decimal Low, decimal Close> object would be nice. Thanks.
